README.md

```diff
@@ -55,3 +55,14 @@ present in Subworkflow units in the `WoDe` package ecosystem. The entities provi
 - `Method` - See [Method Overview](https://docs.mat3ra.com/methods/overview/) for more details
 - `Model` - See [Model Overview](https://docs.mat3ra.com/models/overview/) for more details
 
+## Model Tree
+The `MoDe` package contains asset files defining a _model tree_, whereby each file
+defines a parts of the resulting model data. During `postinstall`, these files are processed
+by the `build_model_tree.js` script and a single model tree is created.
+The file path to a given asset file also determines its position in the tree using
+the following rules:
+- The asset folder `assets` represents the root node.
+- Asset files are expected to have the `.yml` file extension.
+- Each asset file defines a node.
+- Each directory having the same basename as an asset file defines the children of the corresponding node.
+- Asset file names ending in `.nodes.yml` define multiple children at once.
```

Comment:
1. Should we add a reference to CateCom? The purpose of the tree is not clear from what's written at present.
2. Lines 49, 50 should be deleted and the following explanation 52-56 should be added to line 6 at the top